client/store/user.js

```diff
@@ -6,6 +6,8 @@ import history from "../history"
  */
 const GET_USER = "GET_USER"
 const REMOVE_USER = "REMOVE_USER"
+const ISADMIN_USER = "ISADMIN_USER"
+const GET_ALL_USERS = "GET_ALL_USERS"
 
 /**
  * INITIAL STATE
```

Comment:
Your defaultUser structure should contain space for both the adminUser themselves, and the allUsers array that they will be able to look at, so
```
const defaultUser = { user: { }, allUsers: [ ] }